src/java.compiler/share/classes/javax/annotation/processing/Processor.java

```diff
@@ -195,7 +195,7 @@
  * processors can operate on files.
  *
  * @apiNote Implementors of this interface may find it convenient
- * to extend {@link AbstractProcessor} rather than implementing this
+ * to extend {@link AbstractProcessor} rather than implement this
```

Comment:
The original grammar was correct: "rather than something" so "implementing" is correct as a noun. Or you can say "to implement".

Reply:
I will admit a higher than average fondness for gerunds ;-)

Reply:
> The original grammar was correct: "rather than something" so "implementing" is correct as a noun. Or you can say "to implement".

To me, I read it like this: "to extend rather than implement". But if Joe thinks like you do, I will revert this.

Reply:
Since Dan has [replied] on "type" vs "interface", is "implement" vs "implementing" the only place where we disagree? Joe, do you want me to revert it to "implementing"?

[replied]: https://github.com/openjdk/jdk/pull/20937/files/ab574c569166f3200782f4c5e02ae69afddfada4#r1759003606

Reply:
I'd prefer "implementing" was kept.